include/commissioner/commissioner.hpp

```diff
@@ -1146,6 +1146,39 @@ class Commissioner
      *         git repository.
      */
     static std::string GetVersion(void);
+
+    /**
+     * @brief Asynchronously requests diagnostic TLV data from a Thread device.
+     *
+     * This method sends a DIAG_GET.req message to the specified Thread device,
+     * requesting the set of diagnostic data indicated by `aDiagTlvFlags`.
+     * The response, or any errors encountered, will be delivered to the provided `aHandler`.
+     *
+     * @param[in, out] aHandler     A handler to process the response or any errors.
+     *                              This handler is guaranteed to be called.
+     * @param[in]     aAddr         Mesh local address (mesh local prefix + RLOC) of the target Thread device.
+     * @param[in]     aDiagTlvFlags Bitmask specifying the desired diagnostic TLVs.
```

Comment:
Why don't we use a list of TLV types directly? Are those TLV types publicly available as constants in `commissioner/include/`, otherwise, how would you expect developers to easily know what TLV types are available?

Reply:
Thanks, updated.